review_score_app/ServiceLayer/sentiment_api.py

```
@@ -0,0 +1,24 @@
+import requests
+import json
+ 
+
+def sentiment_score(text):
+	url = 'http://api.text2data.com/v3/analyze'
+	payload = {
+	    'DocumentText': text, 
+	    'IsTwitterContent': 'false',
+	    'PrivateKey': '08661BB6-5DFA-4630-9455-05F60C3E0A9D', #add your private key here (you can find it in the admin panel once you sign-up)
+	    'Secret':'', #this should be set-up in admin panel as well
+	    'RequestIdentifier': '' #optional, used for reporting context
+	}
+	 
+	r = requests.post(url, data=payload)
+	data = r.json()
+	  
+	if data['Status'] != 1: 
+		print(data['ErrorMessage'])
```

Don't print value randomly in the code. Worst case, create a "logger" class and just invoke it to log information. At this point it's fine to print from there, but if in the future you decide you want to log things differently you don't have to go through all your code and "replace prints".

Also if something goes wrong you want to throw an error here rather than continue regular execution - or of course you have the option to allow the operation to continue in this situation.